src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -65,6 +65,10 @@ const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
   const isRequestAllowed =
     changeRequest.submitter.userId === currentUser.userId && changeRequest.status === ChangeRequestStatus.Open;
 
+  function potentialCrReviewers(value: UserWithScheduleSettings): value is UserWithScheduleSettings {
```

Comment:
Can you make this a `const` arrow function so you shouldnt need the `is` type assertion and you dont have to use an arrow function in the filter